README.md

```diff
@@ -28,6 +28,6 @@ we clarify here for external readers:
 
 The documents in this project are licensed under the CC-By 3.0 License, which
 encourages you to share these documents. See
-https://creativecommons.org/licenses/by/3.0/ for more details.
+https://creativecommons.org/licenses/by/3.0/ formed more details.
```

Comment:
Nice attempt.
Please, note that the essence of contributing to projects is to improve/add more value. You changed "for" to "formed" making the sentence wrong.
You can simply add a simple sentence.

Reply:
Okay . Thanks